js/src/Ice/ConnectionI.js

```diff
@@ -1212,13 +1177,10 @@ export class ConnectionI {
                 // Otherwise, prepare the next message stream for writing.
                 //
                 message = this._sendStreams[0];
-                Debug.assert(!message.prepared);
```

Comment:
Removed `message.prepared` which was only read by the asserts. Will fix in other mappings too.